sled-agent/src/metrics.rs

```diff
@@ -84,6 +87,8 @@ pub(crate) enum Message {
     TrackOptePort { zone_name: String, name: String },
     /// Stop tracking the named OPTE port.
     UntrackOptePort { name: String },
+    /// Notify the task that a sled has been synced.
+    SyncedSled { sled_id: Uuid, synced: bool },
```

Comment:
I would rename this variant to `TimeSynced` or something like that, to be more clear _what_ has been synced.